### PR TITLE
chore: include boost stuff in the changelog

### PR DESCRIPTION
**Pull request by GabrielBuragev**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,26 +7,35 @@ intended to be used with.
...
 which it is deprecated.
 
-## Deprecation warnings
-
-### 1.3.0
-
-- `getQuote` can respond with a 500 status code and a JSON response body with an
-  string `error` field. This field has been replaced by `message` to be inline
-  with how errors are returned in other parts of the API.
-
```
Comment (GabrielBuragev) [inline CHANGELOG.md:17]: Deduped as it is already included under the `## 1.3.0` section,.

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,26 +7,35 @@ intended to be used with.
...
 - `SwapSDK.prototype.getStatus` will return the list of affiliate brokers for a
   deposit channel if it was opened with affiliates.
 - `SwapSDK.prototype.getQuote` supports an optional `brokerCommissionBps` and
   `affiliateBrokers` option. If given, the `brokerCommissionBps` option will be
   used instead of the `brokerCommissionBps` used to initialize the SDK instance.
-- `SwapSDK.prototype.requestDepositAddress` supports an optional `brokerCommissionBps`
-  and `affiliateBrokers` option. The new options are only available when initializing
+- `SwapSDK.prototype.getQuote` now returns a `boostQuote` property whenever available for
+  the requested route and amount. Currently only available for `BTC -> Any` routes.
+- `SwapSDK.prototype.requestDepositAddress` supports an optional `brokerCommissionBpsş
```
Comment (niklasnatter) [inline CHANGELOG.md:32]:
```suggestion
- `SwapSDK.prototype.requestDepositAddress` supports an optional `brokerCommissionBps`
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -7,26 +7,35 @@ intended to be used with.
...
-
-- `getQuote` can respond with a 500 status code and a JSON response body with an
-  string `error` field. This field has been replaced by `message` to be inline
-  with how errors are returned in other parts of the API.
-
 ## 1.4.0
 
 ### Added
 
+- `SwapSDK.prototype.getStatus` now returns a bunch of properties used to identify
```
Comment (niklasnatter) [inline CHANGELOG.md:14]:
```suggestion
- `SwapSDK.prototype.getStatus` now returns multiple of properties used to identify
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -7,26 +7,35 @@ intended to be used with.
...
-- `SwapSDK.prototype.requestDepositAddress` supports an optional `brokerCommissionBps`
-  and `affiliateBrokers` option. The new options are only available when initializing
+- `SwapSDK.prototype.getQuote` now returns a `boostQuote` property whenever available for
+  the requested route and amount. Currently only available for `BTC -> Any` routes.
+- `SwapSDK.prototype.requestDepositAddress` supports an optional `brokerCommissionBpsş
+and `affiliateBrokers` option. The new options are only available when initializing
   the SDK with a brokerUrl and will be applied only to the requested deposit channel.
+- `SwapSDK.prototype.requestDepositAddress` now supports creating boostable channels by
+  setting the boost fee bps limit that the user is willing to tolerate. By passing the
+  `maxBoostFeeBps` option to this method, one can set the boost fee limit in bps for that
```
Comment (niklasnatter) [inline CHANGELOG.md:37]:
```suggestion
  `maxBoostFeeBps` option to this method, one can set the maximum boost fee in bps for that
```

**Review by niklasnatter — APPROVED** [review]

**Pull request merged**